wheatley/bot.py

```diff
@@ -262,32 +259,23 @@ def tick(self) -> None:
 
         self._place += 1
 
-        if self._place == self.number_of_bells:
+        if self._place >= self.number_of_bells:
```

Comment:
If the number of bells in the tower reduces mid row we might as well gracefully handle it instead of keep increasing `self._place` until we reach unknown bells

Reply:
Oh that's a very good point - I hadn't considered that as a possibility.